introduction.md

```diff
@@ -1,3 +1,9 @@
 This is some weird code and function mames.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?